Sandbox/nona

nona1314 opened the pull request:
I added function can retrieve files from sandbox. I think its parameter (sandbox path) is quite tricky so I added comment on that function.

Another change is file copy functions. I changed their name and parameters to use list of file name and bytes pair.

nona1314 committed:
chage file copy functions

'container' in function name chages to 'sandbox'

byte array copy function chnags to multiple file writing functions
using list of (file name, file contents in bytes) pair, you can write
file in sandbox.

nona1314 committed:
test_sandbox path typo err

nona1314 committed:
add get_files_from_sandbox function

using path (in sandbox), you can get files in that directory.

wowoto9772 reviewed (commented):
write_files_in_sandbox 메소드에서 files 파라미터를 주석 말고, 명시 할 수 있는 방법은 없을까?

get_file_from_sandbox 메소드에서 해당 경로에 파일이 존재하는지 확인이 필요할 듯

test_sandbox.py 에
호스트의 여러 파일을 샌드박스로 옮기는 것도 추가하면 좋을 듯

nona1314 committed:
change get_files test to send_and_get_files test

becuase get_file test acutally do is send multiple files to sandbox then
retrieve those file to host.

add parameter type in write_files_in_sandbox
